dotcom-rendering/src/web/components/AdSlot.tsx

```diff
@@ -96,6 +96,19 @@ const merchandisingAdStyles = css`
 	min-height: 250px;
 `;
 
+const mostpopAdStyles = css`
```

Comment:
`mostPopAdStyles`?

Reply:
I started off with `mostPopAdStyles` but went with `mostpopAdStyles` in the end to align with the value of `position` (`mostpop`).

But I'm not strong here at all and I agree that consistent camel case is better so I'll make the change. Arguably we should also change `position` to match...

Reply:
Done. I briefly looked at updating `position` but the string `mostpop` is also used as part of a classname and we really don't want to be messing with css classname conventions that bleed into FE and commercial code 😬